dvc/tree/azure.py

```diff
@@ -29,21 +30,77 @@ class AzureTree(BaseTree):
     def __init__(self, repo, config):
         super().__init__(repo, config)
 
+        self._account_url = None
         url = config.get("url", "azure://")
         self.path_info = self.PATH_CLS(url)
 
         if not self.path_info.bucket:
             container = self._az_config.get("storage", "container_name", None)
             self.path_info = self.PATH_CLS(f"azure://{container}")
```

Comment:
what happens if container is None here? should we throw an exception?

Reply:
I'm not entirely sure, I'll try have to try it out because I can't tell exactly from the code.